pkg/provider/provider.go

```diff
@@ -170,7 +175,7 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
 	}
 
 	// Feature flags to enable
-	if testing {
+	if preview_features {
```

Comment:
Just seeing this block now! I don't think you want this to be part of the TF provider itself? I think you just want the test script to enable these things:

```
environmentd \
  --system-parameter-default=enable_webhook_sources=true \
  --system-parameter-default=enable_connection_validation_syntax=true
```

You can also just use the `--all-features` flag if you'd like to unconditionally enable all feature flags, without manually managing the list.

Reply:
Oh neat! Just discussed this with @dehume and updated it accordingly.